app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -70,6 +70,14 @@ define([
         window.confirm.restore && window.confirm.restore();
       });
 
+
+      it('it displays a message instead of an empty table, if there are undefined active tasks', function () {
+        spy = sinon.spy(Actions, 'setCollection');
```

Comment:
good point though, it's unclear what's happening.
I can initialize the spy in beforeEach, and then each test will have to assign it again, if necessary.